### PR TITLE
Added JS tutorial 3 text

### PR DESCRIPTION
**Pull request by SHeimlich**
Note: this is the first tutorial whose order is being changed. Hence the folder name change.

### CODE REVIEW

**Comment by Cypher1** [comment]
Looks good @SHeimlich !

File: particles/Tutorial/Javascript/README.md [hunk 1]
```diff
@@ -94,3 +94,73 @@ defineParticle(({SimpleParticle, html}) => {
 ```
 
 Now your code should say “Hello, Human!”. You can update this by changing what `render()` returns. In upcoming tutorials, we will see how this can be updated based on user input.
+
+# Slots: The root of the matter
```
Comment (piotrswigon) [inline particles/Tutorial/Javascript/README.md]: Well done :D

File: particles/Tutorial/Javascript/README.md [hunk 2]
```diff
@@ -94,3 +94,73 @@ defineParticle(({SimpleParticle, html}) => {
...
+
+
+// Unlike previous recipes, this one includes two particles.
+recipe RenderSlotsRecipe
+  ParentParticle
+    // The ParentParticle consumes root just like particles in previous examples.
+    consume root
+      // ParentParticle also provides mySlot. Note the additional tab over.
+      // The name "slot" is the name the recipe uses for the slot, and is how
+     // the recipe connects the parent's mySlot and the child's mySlot
```
Comment (piotrswigon) [inline particles/Tutorial/Javascript/README.md]: indent is off

File: particles/Tutorial/Javascript/README.md [hunk 3]
```diff
@@ -94,3 +94,73 @@ defineParticle(({SimpleParticle, html}) => {
...
+
+  description `Javascript Tutorial 3: Render Slots`
+```
+
+Next, we implement the parent and child particles in Javascript. To do this, we will create a Javascript file for each particle as outlined in the Arcs manifest file. We start with the ParentParticle which provides the slot.
+```javascript
+defineParticle(({SimpleParticle, html}) => {
+ return class extends SimpleParticle {
+   get template() {
+     // The parent particle needs to provide a div with slotid "mySlot". This is where the child particle will be rendered.
```
Comment (piotrswigon) [inline particles/Tutorial/Javascript/README.md]: Very long comment... split into 2 lines?

**Pull request merged**